src/modules/task-dynamic-ui/task-dynamic-ui.ts

```diff
@@ -128,18 +147,18 @@ export class TaskDynamicUi {
     const identity: IIdentity = this._getIdentity();
 
     try {
-      const correlationNotGiven: boolean = this._correlationId === undefined;
-      const processModelIdNotGiven: boolean = this._processModelId === undefined;
+      const correlationNotGiven: boolean = this.correlationId === undefined;
+      const processModelIdNotGiven: boolean = this.processModelId === undefined;
```

Comment:
IMO `noProcessModelId` sounds nicer
```suggestion
      const noProcessModelId: boolean = this.processModelId === undefined;
```